spec/core_functions/string/split.hrx

```diff
@@ -0,0 +1,179 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> empty/input.scss
+a {b: split("")}
+
+<===> empty/output.css
+a {
+  b: [""];
+}
+
+<===>
+================================================================================
+<===> separator/input.scss
+a {b: split("a, b, c", ", ")}
+
+<===> separator/output.css
+a {
+  b: ["a", "b", "c"];
+}
+
+<===>
+================================================================================
+<===> empty_separator/input.scss
+a {b: split("Helvetica", "")}
```

Comment:
Probably also a good idea to test an empty string with a non-empty separator

Reply:
Added a test for an empty string and a non-empty separator.